src/views/List.jsx

```diff
@@ -38,7 +38,16 @@ export function List({ data }) {
 			</form>
 			<ul>
 				{filteredItems.map((item) => {
-					return <ListItem key={item.id} name={item.name} />;
+					return (
+						<ListItem
+							key={item.id}
+							name={item.name}
+							itemId={item.id}
+							listPath={listPath}
+							purchaseDate={item.dateLastPurchased}
+							nextPurchaseDate={item.dateNextPurchased}
```

Comment:
We don't need to pass this prop, as discussed [here](https://github.com/the-collab-lab/tcl-75-smart-shopping-list/pull/39#discussion_r1747834411) and [here](https://github.com/the-collab-lab/tcl-75-smart-shopping-list/pull/39#discussion_r1747839551)!